system/Validation/Rules.php

```diff
@@ -436,6 +436,47 @@ public function required_without(
         return true;
     }
 
+    /**
+     * This field is required when any of the other required fields have their expected values present
+     * in the data.
+     *
+     * Example (The special_option field is required when the normal_option,1,2 field has the given value.):
```

Comment:
This error message is confusing:

The special_option field is required when the normal_option,1,2 field has the given value.

The other field is `normal_option`, not `normal_option,1,2`.

Reply:
I thought about that too, but the fields and values ​​are combined in the {param} placeholder. How to extract fields and values ​​from {param}? At least we can separate them and differentiate between fields and values then provide proper error messages. do you have any ideas?